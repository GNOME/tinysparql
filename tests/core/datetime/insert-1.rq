SELECT ?s ?v
WHERE {
	?s example:p ?v
}
ORDER BY ?x
