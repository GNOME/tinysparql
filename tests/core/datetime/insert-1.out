"http://example/x"	"0100-12-31T21:00:00-03:00"
"http://example/y"	"0100-12-31T21:00:00.123321-03:00"
"http://example/z"	"0100-12-31T21:00:00.123321Z"
